### ISSUE TITLE
Create a go.mod

### ISSUE THREAD

**Issue by andradei** [natefinch/lumberjack#65]
To make this package module-aware.

**Comment by natefinch** [comment]
Yeah, I should do that. Thanks for the reminder.  I'll get that done soon.

**Comment by xiegeo** [comment]
Right now the package imports fine by `go mod`, it shows as [gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3](https://github.com/xiegeo/thuder/blob/9e1703ae723ac3677b8c1088d54e9914f865385d/go.mod#L10) in my downstream project.

The only problem is that the tag v2.1 is not [Semver](https://github.com/kyokomi/emoji/issues/23), so it probably needs a new v2.1.1 tag for it to show as `v2.1.1` instead of `v2.0.0-20170531160350-a96e63847dc3`.

There are no build dependencies, however adding a go.mod file will help with defining test dependencies.
